git-tree.py

```diff
@@ -190,10 +208,6 @@ def print_table(print_outs, branches):
         deltas = ahead + ":" + behind
         deltas_column_length = len(branch_ahead_str) + len(branch_behind_str) + 3
 
-        if assume_main_is_upstream(branch.upstream_branch):
```

Comment:
Will skip deltas for the root node and/or the main branch as discussed

Reply:
Update logic so it doesn't show deltas for main branch.